crates/ironrdp-pdu/src/basic_output/fast_path.rs

```diff
@@ -289,6 +333,9 @@ pub enum FastPathError {
     UnsupportedFastPathUpdate(UpdateCode),
     #[error("Invalid RDP Share Data Header: {0}")]
     InvalidShareDataHeader(String),
+    // TODO(@pacmancoder): Temporary solution
+    #[error(transparent)]
+    Pdu(#[from] PduError),
```

Comment:
@CBenoit A quick question:
This module is still using legacy parsing traits, so we have a few possibilities here:
- Refactor the fast path error handling and switch to `PduError` instead of the legacy module-specific `FastPathError` (I lean to this solution the most, just a bit more effort)
- Use legacy parsing traits and keep fast path module API the same
- Encapsulate `PduError` inside legacy error as I did temporarily here (I'll add `Box` around the error for good measure if we go with this plan)

What we should do in such cases? I want to hear your opinion on that

Reply:
I’m all in refactoring away from legacy traits as we continue moving forward with new features. :+1: 

Of course, it’s also fine to continue using the legacy parsing traits for now if this is too much work all at once.